### ISSUE TITLE
[Feat]Allow Users to Share a specific task 

### ISSUE THREAD

**Issue by Sanat-Jha** [IMGIITRoorkee/Taskly#81]
**Describe the solution you'd like**
Users can share a task and all its details to someone, through any platform like whatsapp, email etc.

**Comment by sommye-ctr** [comment]
I think it would be best to share the link of the task. It would be in support of #46

**Comment by Sanat-Jha** [comment]
agreed if #46 gets resolved then an option of link would be better. The link can just open a box of info about task similar to #74 .

**Comment by sommye-ctr** [comment]
As per #57 I just pushed some changes that were requested. Currently as per implementation, the link is opening the task edit screen (as requested by maintainers on discord). If they request it to open the specific dialog box, changes then the info box can be opened too.
